astrodb_utils/fits.py

```diff
@@ -16,7 +16,7 @@
     "check_header",
 ]
 
-logger = logging.getLogger("astrodb_utils")
+logger = logging.getLogger(__name__)
```

Comment:
I sometimes can't get `__name__` to work, but if it works in this case- that's great.